pipelines/LATISS/ApPipe.yaml

```diff
@@ -2,3 +2,16 @@ description: AP pipeline specialized for LATISS
 instrument: lsst.obs.lsst.Latiss
```

Comment:
I recommend updating the description to mention that this pipeline is optimized for Prompt Processing, especially if the comments use PP's limitations to justify the config overrides.